public/js/main.js

```diff
@@ -214,26 +181,45 @@ $(document).ready(function() {
     /**
      * Get Category and send ajax request to server
      */
-    $selectCate = document.getElementById("select-category");
     $('#select-category').change(function (e) {
-        //set page of paginate = 1
-        $i = 1;
         $('#select-food').empty();
-        $('#choose-food').html($('#choose-food').attr("data-text"));
-        if (window.location.href.indexOf("create") > 0) {
-            $url = window.location.href;
-        } else {
-            $url = window.location.href + "/create";
-        }
+        $url = '/daily-menus/get-list-food';
```

Comment:
Don't hard code URL, please get from a route. You can define from master layout and call in js file